photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -402,7 +402,7 @@ object Driver {
         val logsDir = new Path(params.outputDir, LOGS).toString
         implicit val logger = new PhotonLogger(logsDir, sc)
         // TODO: This Photon log level should be made configurable
-        logger.setLogLevel(PhotonLogger.LogLevelInfo)
+        logger.setLogLevel(PhotonLogger.LogLevelDebug)
```

Comment:
If debug level doesn't incur too much penalty when looking for root cause of issues in prod by mining the logs, I'm OK with it. My concern is to not generate a log that is so big that it: degrades runtime performance ; crashes some space quota on the grid ; makes it hard to figure out issues because you have to grep through multiple TBs of log files on remote machines.